doc/nrf/releases_and_maturity/known_issues.rst

```diff
@@ -75,7 +75,7 @@ KRKNWK-19647: nRF53 has high power consumption in sleep mode
 
   **Affected platforms:** nRF5340
 
-.. rst-class:: v2-9-0-nRF54H20-rc1 v2-9-0 v2-8-0
+.. rst-class:: v2-9-0-nRF54H20-1-rc2 v2-9-0 v2-8-0
```

Comment:
rc1 and rc2 are two different releases, shouldn't both be mentioned?

Reply:
Added both the versions.